.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Please, consider to add `node_modules` into `.gitignore` file

Reply:
node_modules is not included in this project. Thanks for the review